app/config/config.yml

```diff
@@ -7,6 +7,7 @@ imports:
     - { resource: services/cms.xml }
     - { resource: services/community.xml }
     - { resource: services/external_services.xml }
+    - { resource: services/feed_generator.xml }
```

Comment:
Can you put this service in `services/cms.xml` instead, to avoid too much services files?